packages/expo-dev-launcher/package.json

```diff
@@ -35,7 +35,7 @@
     "semver": "^7.3.5"
   },
   "devDependencies": {
-    "@react-navigation/bottom-tabs": "5.11.15",
```

Comment:
I think this might have been an unintentional change

Reply:
this reflects the state of main

Reply:
I think I pushed this change to this branch (not main) in my patch to `react-navigation`